bin/do/setup

```diff
@@ -0,0 +1,263 @@
+#!/bin/sh
+
+# On Server
+# -------------------------------------------------
+GIT_URL="$1"
+HOST="$2"
+MONGO_PASS=$(date +%s | sha256sum | base64 | head -c 32 ; echo)
+
+sudo apt-get update
```

Comment:
Why are you using sudo here an below? You're logged in as root, so no need in `sudo` which is `super user do` = `root do`.